app/template/page.client.tsx

```diff
@@ -0,0 +1,187 @@
+'use client';
+
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import PageHeading from '../_components/utility/pageHeading';
+import getFromApiClientSide from './api';
+import layoutConfig from '../_config/layout';
+import GetColor from '../_hooks/colorSelector';
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  Button,
+  Center,
+  Flex,
+  List,
+  ListItem,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react';
+
+export interface IServerDataSchema {
+  commodityName: string;
+  displayName: string;
+  type: string;
+  isRare: boolean;
+}
+
+interface Props {
+  serverData: IServerDataSchema[];
+}
```

Comment:
If we can keep some comments in the template file where it clarifies what is the purpose of that component e.g., here I believe the dev has to replace or remove this depending on the data type of the response. Totally up for debate if you think we should have comments in the file where relevant.

Reply:
done